kolibri/core/assets/src/views/k-textbox/index.vue

```diff
@@ -107,6 +107,13 @@
         type: Boolean,
         default: false,
       },
+      /**
+       * Whether or not to display as a floating label
+       */
+      floatingLabel: {
+        type: Boolean,
+        default: true,
+      },
```

Comment:
Note that `k-textbox` is a core component in the styleguide, and we need to be judicious about adding new props.

I'm going to make this prop `private` so it doesn't increase the API surface area.